it/distribution_test.py

```diff
@@ -50,5 +50,5 @@ def test_docker_distribution(cfg):
 def test_does_not_benchmark_unsupported_distribution(cfg):
     port = 19200
     it.wait_until_port_is_free(port_number=port)
-    assert it.execute_test(cfg, f"--distribution-version=\"1.7.6\" --workload=\"{it.WORKLOADS[0]}\" "
+    assert it.execute_test(cfg, f"--distribution-version=\"1.3.8\" --workload=\"{it.WORKLOADS[0]}\" "
```

Comment:
Doesn't need to happen during this PR since it takes a while to run the integration tests, but we could set this as a global variable or read from a config.

Reply:
Just curious, why was this changed 1.3.8?